### PR TITLE
Send interval

### PR DESCRIPTION
**Pull request by mark-meyer**
Adds a delay when sending twilio messages in order to avoid errors. It will currently send about 2/second, but could probably be increased. Also adds basic test for `cron_job.sendmessage()`.

### CODE REVIEW

File: cron_job.js [hunk 1]
```diff
@@ -28,39 +28,42 @@ var job = new cron.CronJob({
...
+        var inter = setInterval( gen => {
+            var next = gen.next()
+            if (next.done)  return clearInterval(inter)
+            var subscriber = next.value
+            module.exports.sendToTwilio(subscriber.phone, randomElement(message_text.NOTIFICATIONS))
+        }, DELAY, subscribers[Symbol.iterator]())
     })
 }
 
+function sendToTwilio(to, body){
```
Comment (NigelKibodeaux) [inline cron_job.js:47]: looks like this doesn't need to accept the body

**Review by NigelKibodeaux — APPROVED** [review]
looks good other than my 1 comment. nice test!

File: cron_job.js (hunk 1 above)
Comment (mark-meyer) [reply cron_job.js:47]: Thanks for catching that -- the `to:` is also wrong. I think it makes sense to pass the body in, so this function is just responsible for sending, but I'll change it to make it actually use the argument.

Comment (mark-meyer) [reply cron_job.js:47]: I think it's fixed.

**Review by NigelKibodeaux — APPROVED** [review]

**Pull request merged**